dotnet/G3DViewer/MainViewModel.cs

```diff
@@ -239,7 +207,7 @@ internal void BakeInstance(Ara3D.IG3D g3dFile, Ara3D.Matrix4x4 Transform)
             {
                 int faceSize = faceSizes[face];
 
-                List<SharpDX.Vector3> vectors = new List<Vector3>(faceSize);
+                var vectors = new List<Vector3>(faceSize);
                 for (int i = 0; i < faceSize; i++)

```

Comment:
I just noticed `mul()`, this is the same as the `Transform` function already in Ara3D.